From fccc246a3ceacab8d443eb576d0365f9e0ed0521 Mon Sep 17 00:00:00 2001
From: Alex Wilson <alex.wilson@joyent.com>
Date: Fri, 11 Nov 2016 17:53:35 -0800
Subject: [PATCH] joyent/node-cueball#48 use higher DNS concurrency on
 bootstrap Reviewed by: Cody Peter Mello <cody.mello@joyent.com>

---
 lib/resolver.js | 3 ++-
 package.json    | 2 +-
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/lib/resolver.js b/lib/resolver.js
index a5e6562..aa0474f 100644
--- a/lib/resolver.js
+++ b/lib/resolver.js
@@ -309,7 +309,8 @@ CueBallDNSResolver.prototype.state_check_ns = function (S) {
 				service: '_dns._udp',
 				defaultPort: 53,
 				log: this.r_log,
-				recovery: this.r_recovery
+				recovery: this.r_recovery,
+				maxDNSConcurrency: 10
 			});
 			this.r_bootstrap = res.r_fsm;
 			CueBallDNSResolver.bootstrapResolvers[notIp[0]] =
diff --git a/package.json b/package.json
index 5430225..ddd4283 100644
--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "cueball",
-  "version": "1.1.4",
+  "version": "1.1.5",
   "description": "",
   "main": "lib/index.js",
   "dependencies": {
-- 
2.21.0

